astropy/units/astrophys.py

```diff
@@ -145,3 +137,26 @@
 from .utils import generate_unit_summary as _generate_unit_summary
 if __doc__ is not None:
     __doc__ += _generate_unit_summary(globals())
+
+
+# -------------------------------------------------------------------------
+
+def __getattr__(name):
```

Comment:
My tendency would be to do the deprecation warning at the top level and here just remove `littleh` - everything below top-level is nominally private API (though I guess in units this is not strictly true, as `u.si` and `u.cgs` can be used for composition).

But no big deal - what you have is definitely fine too!

Reply:
I was debating this when moving the definitions. While I think it would be fine/better for ``littleh`` in ``astrophysics``, I've seen people use ``u.equivalencies...``, so I wanted to keep at least that deprecation in the equivalencies module.
Also, if we have lightly plan on a timeline for removal... a problem that nicely resolves itself.
The standard is 2 minor versions later, right?

Reply:
Indeed, it will resolve itself, so it is not a big deal.